### DISCUSSION TITLE
Delete old log files

### DISCUSSION THREAD

**Discussion by edbadaaz** [Security-Onion-Solutions/securityonion#9909] [General] [answered]
I have an onion that has received a lot of logs and as a result its hd ran out of available space.

Is there a command that can remove old log files?

![image](https://user-images.githubusercontent.com/126823982/223553367-9fb5d275-d656-4ade-8d12-4bd1e326a677.png)

**Comment by reyesj2** [comment, marked as answer]
Looks like you're running a machine with lower than required storage https://docs.securityonion.net/en/2.3/partitioning.html#minimum-storage. 

Network / host logs & pcap get stored in /nsm 

Looking at your older post #9885 you need to add a drive for storage to this machine and expand your / and ideally /nsm lvms

You can delete your PCAP and logs from /nsm with ```sudo so-elastic-clear && sudo so-nsm-clear```

**Reply by edbadaaz** [reply]
Thanks, but one question.

When performing the command "so-elastic-clear && sudo so-nsm-clear".

Would it remove all content from the directories or just the old ones?

**Reply by reyesj2** [reply]
That will remove all your log data. I would strongly recommend increasing your storage capacity by as much as possible if you're trying to retain any sort of historical logs.

**Reply by edbadaaz** [reply]
Is there any way to limit the log occupancy in onion?

@reyesj2
